stock_move_manual_lot/tests/test_stock_move_manual_lot.py

```diff
@@ -167,7 +185,10 @@ def test_03_multiple_lines_move_lot(self):
              ],
         })
         self.assertEqual(line_without_lot.lot_id, self.lot1)
-        self.assertFalse(line_with_lot.lot_id)
+        other_line = self.picking.move_line_ids - line_without_lot
```

Comment:
Note that with these changes, the `line_with_lot` no longer exists, as it is one of those lines that ends up with product_qty 0 and is thus deleted on purpose and has its stock.move reassigned.